File: Pak Dengklek dan Operasi Sederhana/Pak-Dengklek-dan-Operasi-Sederhana.cpp
#include <iostream>

int main ()
{
    short X = 0, Y = 0;
    std::cin >> X >> Y;

    std::cout << X + Y << "\n";
    std::cout << X - Y << "\n";
    std::cout << X * Y << "\n";
    std::cout << X / Y << "\n";
    std::cout << X % Y;

    return 0;
}

File: Pak Dengklek dan Operasi Sederhana/PakDengklekdanOperasiSederhana.java
import java.util.Scanner;

public class PakDengklekdanOperasiSederhana
{
    public static void main (String [] args)
    {
        Scanner scan = new Scanner (System.in);
        short X = scan.nextShort ();
        short Y = scan.nextShort ();

        System.out.println (X + Y);
        System.out.println (X - Y);
        System.out.println (X * Y);
        System.out.println (X / Y);
        System.out.print (X % Y);

        scan.close ();
    }
}